parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -23,8 +23,8 @@ installContainerRuntime() {
     {{if NeedsContainerd}}
         echo "in installContainerRuntime - KUBERNETES_VERSION = ${KUBERNETES_VERSION}"
         if semverCompare ${KUBERNETES_VERSION} "1.22.0"; then
-            CONTAINERD_VERSION="1.5.5"
-            CONTAINERD_PATCH_VERSION="3"
+            CONTAINERD_VERSION="1.5.9"
```

Comment:
just curious, why do we use k8s 1.22.0 to to differentiate containerd versions?

Reply:
As it's a new containerd minor version (previously it was 1.4.x), we wanted to test its stability with AKS for a subset of versions (paired with 1.22 preview). As the containerd upgrade doesn't seem to have issues, we will eventually change it to containerd 1.5 for all k8s versions.  I've also just messaged you a doc with additional reasons/context for this change.

Reply:
Thanks for your explanation and doc.